stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -437,15 +446,11 @@ def get_search(
         collections: Optional[List[str]] = None,
         ids: Optional[List[str]] = None,
         bbox: Optional[List[NumType]] = None,
+        intersects: Optional[str] = None,
         datetime: Optional[Union[str, datetime]] = None,
         limit: Optional[int] = 10,
-        query: Optional[str] = None,
-        token: Optional[str] = None,
-        fields: Optional[List[str]] = None,
-        sortby: Optional[str] = None,
```

Comment:
What are the reasons for removing these fields?

Reply:
They are used for extensions only and should not be part of this abstract method. Users can implement them in their own subclass if they need to.

Reply:
Makes sense. Thank you.